app/views/admin/calendars/_import.html.erb

```diff
@@ -3,7 +3,11 @@
 
 <%= render 'status' %>
 
-<% if @calendar.calendar_state.idle? %>
+<% if @calendar.calendar_state.idle? || @calendar.calendar_state.bad_source? %>
+  <% if @calendar.calendar_state.bad_source? %>
+    <p class="bigger"> We're unable to read this calendar - the URL may be broken or there may be some invalid data in it.<br/>Please check the calendar is set up correctly and try again.<br/>
+    </p>
```

Comment:
Also this copy may be completely wrong - was trying to phrase this in a non-tech friendly way - but let me know if there's a better turn of phrase